app/models/order.rb

```diff
@@ -51,6 +67,28 @@ def sufficient_funds_for_buy
     errors.add(:shares, "Insufficient funds. You have #{formatted_balance} but need #{formatted_cost}")
   end
 
+  def sufficient_funds_for_sell_when_update
+    # we would want to execute this method only when we are just updating the shares
```

Comment:
We don't want to leave these comments in, can you remove them please?
Let's aim to name the methods and create tests in such a way that we are document what's happening.